test/functional/feature_dersig.py

```diff
@@ -40,7 +40,7 @@ def unDERify(tx):
 class BIP66Test(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
-        self.extra_args = [['-whitelist=noban@127.0.0.1', '-dip3params=9000:9000', '-par=1']]  # Use only one script thread to get the exact reject reason for testing
+        self.extra_args = [['-whitelist=noban@127.0.0.1', '-dip3params=9000:9000', '-par=1', '-vbparams=v20:0:999999999999:480:384:288:5:-1']]  # Use only one script thread to get the exact reject reason for testing
```

Comment:
note for reviewers: if this PR would be merged after https://github.com/dashpay/dash/pull/5658 need to replace `-1` to `0` so far as MNActivationHeight is refactored to flag `useEHF`.
```suggestion
        self.extra_args = [['-whitelist=noban@127.0.0.1', '-dip3params=9000:9000', '-par=1', '-vbparams=v20:0:999999999999:480:384:288:5:0']]  # Use only one script thread to get the exact reject reason for testing
```

Reply:
@knst Since this one was merged, adjust `feature_dersig.py` on #5658